### ISSUE TITLE
A11y: Contrast on Tag Flare

### ISSUE THREAD

**Issue by Link2Twenty** [forem/forem#1273]
**Describe the issue**
The contrast on some of the tags, in post titles, is too low.

**To Reproduce**
It can be hard to see if there are contrast issues if you have good eyes. If you filter the whole site to greyscale it can help you spot issues.

```css
body {
  filter: grayscale();
}
```

![image](https://user-images.githubusercontent.com/3534427/49635936-189e4a80-f9f9-11e8-9853-4a8e24ef1268.png)

**Additional context**
Changing all of the colours to have good contrast values will help boost lighthouse scores.

**Comment by pkfrank** [comment]
Thanks for the issue @Link2Twenty .  I would think this touches on some of the same discussion items from #735.  Appreciate the issue!

**Comment by maestromac** [comment]
@Link2Twenty can you more elaborate on that?

**Comment by Link2Twenty** [comment]
I'm presuming the colour pairings you're using are preset somewhere?
For instance, [ama](https://dev.to/t/ama), which is not compliant, has a background of `#1395B8` and a foreground of `#FFFFFF` meaning it has a contrast of [3.48](https://contrast-ratio.com/#white-on-%231395B8), which is below min spec of 4.5 and way below the recommended 7.

I'm suggesting that during the build process, if there is one, we have something, like my code in #735, that checks the colour combos and outputs a file so someone can look and see the problem areas.

We could compute contrast viable options but a person should do it really.

**Comment by nickytonline** [comment]
Want to take a stab at this for Hacktoberfest @Link2Twenty ?

**Comment by Link2Twenty** [comment]
In admin tags it might be nice to have a message telling the user the contract of their colours. Similar to what I suggested in #735 

Maybe it can be under or beside the preview
![image](https://user-images.githubusercontent.com/3534427/95459511-ad628e00-096b-11eb-9f8a-e9ecd5a33566.png)

Though I don't know if it should be calculated server side or client side. Server side feels like the better option but I wouldn't know where to start with that 😅

**Comment by cmgorton** [comment]
Hey all. Thanks for bringing this up. This issue will now be encompassed in the broader accessibility issue Marcy has opened #11504 . If we have any more comments or suggestions we can leave them in that issue. 
Since that is the case I am going to close this issue.
